src/Common/tests/System/AssertExtensions.cs

```diff
@@ -24,8 +24,18 @@ public static void Throws<T>(string netCoreParamName, string netFxParamName, Act
             string expectedParamName =
                 RuntimeInformation.FrameworkDescription.StartsWith(".NET Framework") ?
                 netFxParamName : netCoreParamName;
-            
-            Assert.Equal(expectedParamName, exception.ParamName);
+
+            if (!PlatformDetection.IsNetNative)
```

Comment:
I would expect this to not build for a lot of projects because I'm almost certain most that include this file don't include the platformdetection.cs file.